addons/sale/report/report_all_channels_sales.py

```diff
@@ -52,7 +52,7 @@ def _so(self):
                     LEFT JOIN product_pricelist pp ON (so.pricelist_id = pp.id)
                     LEFT JOIN uom_uom u on (u.id=sol.product_uom)
                     LEFT JOIN uom_uom u2 on (u2.id=pt.uom_id)
-            WHERE so.state != 'cancel'
+            WHERE so.state in ('sale','done')
         """
```

Comment:
Should include `locked` as well

Reply:
@dbo-odoo that's why I've included 'done'. Locked is `done` state, isn't it?

Reply:
shoot, my bad - in my memory it was still locked and I didn't see it in your list *facepalm*